zap/src/main/java/org/zaproxy/zap/extension/custompages/ExtensionCustomPages.java

```diff
@@ -229,6 +229,13 @@ public void sessionModeChanged(Mode mode) {
         // Ignore
     }
 
+    /**
+     * Tells whether or not the msg matches the specific {@code CustomPageType}
+     *
+     * @param msg the message that will be checked
+     * @param cpType the custom page type to be checked
+     * @return {@code true} if the message matches, {@code false} otherwise
+     */
     public boolean isCustomPage(HttpMessage msg, CustomPageType cpt) {
```

Comment:
- Should we check contexts whether the user requests it or not?
- Does this while implementation actually need to be an "Option" panel instead of context based? (More like Global Excludes?)
- How do we make this pan out for pscan (only if the user has select "Scan only in scope"?)
- Is there a way (I didn't see it) in HostProcess (or whatever) to check isScanningContext? For which if we were only applying CustomPages to context scanning we could bail early here and return false? (isInScope?)